python/langsmith/client.py

```diff
@@ -1288,6 +1262,74 @@ def _hide_run_outputs(self, outputs: dict):
             return outputs
         return self._hide_outputs(outputs)
 
+    def _batch_ingest_ops(
```

Comment:
```suggestion
    def _batch_ingest_run_ops(
```

If this only actually is intended to handle runs